modules/deepintentBidAdapter.js

```diff
@@ -98,6 +99,20 @@ export const spec = {
       deepSetValue(openRtbBidRequest, 'regs.ext.gdpr', (bidderRequest.gdprConsent.gdprApplies ? 1 : 0));
     }
 
+    // GPP Consent
+    if (bidderRequest?.gppConsent?.gppString) {
+      deepSetValue(openRtbBidRequest, 'regs.gpp', bidderRequest.gppConsent.gppString);
+      deepSetValue(openRtbBidRequest, 'regs.gpp_sid', bidderRequest.gppConsent.applicableSections);
+    } else if (bidderRequest?.ortb2?.regs?.gpp) {
+      deepSetValue(openRtbBidRequest, 'regs.gpp', bidderRequest.ortb2.regs.gpp);
+      deepSetValue(openRtbBidRequest, 'regs.gpp_sid', bidderRequest.ortb2.regs.gpp_sid);
+    }
+
+    // coppa compliance
+    if (config.getConfig('coppa') === true) {
```

Comment:
Can you please get this from the request?

Reply:
@ChrisHuie coppa compliance looks to be read from config across adapters hence ensured this is read from config for consistency. Should I still go ahead and make the change ?

Reply:
@parthshah51999 yes please. The majority of adapters are currently reading from config but Prebid is trying to encourage not using config for "coppa" and treating it more similarly to other regs in Prebid. We recently updated our docs on coppa -> https://docs.prebid.org/dev-docs/bidder-adaptor.html#prebid-standard-parameter-locations

Reply:
@ChrisHuie updated the PR. Could you please go through ?